src/util.rs

```diff
@@ -100,7 +109,24 @@ pub fn start_daemon<I, S>(program: &str, args: I) -> io::Result<()>
         I: IntoIterator<Item = S>,
         S: AsRef<OsStr>,
 {
-    Command::new(program).args(args).spawn().map(|_| ())
+    // As per issue 2128 the decision was made to make this call as
+    // similar to the Unix behaviour as possible.
+    //
+    // Setting all the I/O handles to null and setting the
+    // CREATE_NEW_PROCESS_GROUP and CREATE_NO_WINDOW has the effect
+    // that console applications will run without opening a new
+    // console window.
+    //
+    // If visible output is desirable the DETACHED_PROCESS flag could
+    // be used instead of CREATE_NO_WINDOW.
```

Comment:
While this might be a beneficial comment if this is planned to change in the future, I would recommend against putting it here since the plan is to keep things the way they are.

If it is desired to change this in the future, looking into the Windows documentation or this PR should be the first target anyways. This way we can keep the code comments to the essentials and don't have to waste time with info that is likely irrelevant.